robot/rospackages/src/ping_acknowledgment/scripts/ping_response_client.py

```diff
@@ -47,3 +52,5 @@ def usage():
 
     diff = received - sent
     print("Latency: " + str(diff.total_seconds() * 1000) + " ms")
+    ping = rospy.Publisher('/ping_value', Float32, queue_size=10, latch=True)
+    ping.publish(Float32(diff.total_seconds() * 1000))
```

Comment:
So this is not quite what was intended. What I mean is the server should just return the amount directly. Because right now what you have to do is send the ping and then subscribe, we need to be able to do

`rosrun ping_ack ping_ack_client` and just get a number as a result. Rn we get a shitty string like this

```
Warning: no message argument, sending 'silence' to ping response server
usage: /home/cedric/Programming/robotics-prototype/robot/rospackages/src/ping_acknowledgment/scripts/ping_response_client.py [msg]
Pinging server with ping_acknowledgment service
2020-05-27T18:38:14-44
---
Response: To your 'silence', I say 'suh dude'
2020-05-27T18:38:14-44
Latency: 3.799 ms
```
It's completely useless and hard to parse, and to get the actual float value you would need to be subscribed to a topic. But even if you are subscribe to said topic, **you can't tell if you're the one who initiated the ping**